stmhal/usbdev/class/inc/usbd_cdc_msc_hid.h

```diff
@@ -20,6 +20,12 @@
 #define CDC_OUT_EP    (0x03)
 #define CDC_CMD_EP    (0x82)
 
+// Need to define here for usbd_hid_interface.c (it needs HID_OUT_EP_WITH_CDC)
+#define HID_IN_EP_WITH_CDC (0x81)
+#define HID_OUT_EP_WITH_CDC (0x01)
+#define HID_IN_EP_WITH_MSC (0x83)
+#define HID_OUT_EP_WITH_MSC (0x03)
```

Comment:
I used the approach used in usbd_cdc_interface (exposing defines via .h file), but apporach 2) makes much more sense. Will update the PR.